source/security/private-endpoints.txt

```diff
@@ -215,20 +290,18 @@ connection using either the App Services UI or the Admin API.
                  -d '{
                    "cloud_provider_region": "<your-cloud-deployment-region>",
                    "cloud_provider_endpoint_id": "<your-vpc-endpoint-id>"
+                   "endpoint_type":"sync"
                  }'
+            
+            #. You must also create a General Private Endpoint by calling
+               the ``Create a VPC Private Endpoint`` Admin API endpoint again.
+               This time, set the ``endpoint_type`` to ``"general``.
```

Comment:
For technical reviewer: is this true? Or does hitting that Admin API endpoint automatically create the General Endpoint too?

Reply:
Yeah, I think pointing them to the general endpoint section makes the most sense. I did that at first, then changed it to what you saw. Can't remember why.

And thanks for noticing the odd rendering. I fixed that for the general endpoint section, but forgot to update here. Should be good now!